eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -25,12 +25,13 @@ public static function dumpAssets(CommandEvent $event)
         $options = self::getOptions($event);
         $appDir = $options['symfony-app-dir'];
         $webDir = $options['symfony-web-dir'];
-        $env = isset($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : '';
+        // @deprecated: Use of ezpublish-asset-dump-env is deprecated, use standard SYMFONY_ENV instead.
```

Comment:
We don't, but script still stalls and asks for it. We also use vagrant so over there it wouldn't be a problem, but some run eZ on local machines (I run 7 instances at all times, both in prod and dev and reinstall on daily bases so requiring), so I use `ezpublish-asset-dump-env` to just dump for prod, without asking.